Compilation file with Esp8266 3.1.0

After latest ESP8266 platformio update ESPAsyncWiFiManager fails to compile

```
.pio\libdeps\xxxxx\ESPAsyncWiFiManager\ESPAsyncWiFiManager.cpp: In member function 'void AsyncWiFiManager::resetSettings()':
.pio\libdeps\xxxxx\ESPAsyncWiFiManager\ESPAsyncWiFiManager.cpp:693:28: error: no matching function for call to 'ESP8266WiFiClass::disconnect(bool, bool)'
  693 |   WiFi.disconnect(true,true);
      |                            ^
In file included from C:\Users\gmartin\.platformio\packages\framework-arduinoespressif8266\libraries\ESP8266WiFi\src/ESP8266WiFi.h:34,
                 from .pio\libdeps\esp8266-SmartSwitchController\ESPAsyncWiFiManager\ESPAsyncWiFiManager.h:18,
                 from .pio\libdeps\esp8266-SmartSwitchController\ESPAsyncWiFiManager\ESPAsyncWiFiManager.cpp:14:
```

I'll add a pull request with that line enclosed with an `#ifdef`.

--- a/ESPAsyncWiFiManager.cpp
+++ b/ESPAsyncWiFiManager.cpp
@@ -689,9 +689,13 @@ void AsyncWiFiManager::resetSettings() {
   DEBUG_WM(F("THIS MAY CAUSE AP NOT TO START UP PROPERLY. YOU NEED TO COMMENT IT OUT AFTER ERASING THE DATA."));
 	
   WiFi.mode(WIFI_AP_STA); // cannot erase if not in STA mode !
-  WiFi.persistent(true);
-  WiFi.disconnect(true,true);
-  WiFi.persistent(false);	
+  WiFi.persistent (true);
+#if defined(ESP8266)
+  WiFi.disconnect (true);
+#else
+  WiFi.disconnect (true, true);
+#endif
+  WiFi.persistent (false);
 	
   //delay(200);
 }

